app/models/ban.rb

```diff
@@ -35,7 +35,8 @@ class Ban < ActiveRecord::Base
 
   validate :validate_type
   validate :validate_ventban
-  validates_format_of :steamid, with: /\A([0-9]{1,10}:){2}[0-9]{1,10}\Z/, allow_blank: true
+  validates_length_of :steamid, maximum: 14, allow_blank: true
+  validates_format_of :steamid, with: /\A0:[01]:[0-9]{1,10}\Z/, allow_blank: true
```

Comment:
Line is too long. [81/80]<br>Prefer the new style validations `validates :column, format: value` over `validates_format_of`.